src/style.css

```diff
@@ -0,0 +1,363 @@
+}
+#left-menu main > div:last-child:active {
+    background-color: rgba(255, 166, 0, 0.6);
+    color: white;
+}
+
+
+/*STYLES THE NEWSFEED*/
+#post-container {
+    border-left:  1px solid grey;
+    border-right:  1px solid grey;
+    padding: 10px 0 0 0;
+    height: calc(100vh - 10px);
+    min-width: 400px;
+}
+
+#post-container > header {
+    border-bottom: 1px solid grey;
+    position: absolute;
+    width: 100%;
+    top: 10px;
+}
+#post-container > main {
+    display: grid;
+    grid-template-rows: 120px auto;
+}
+
+/*top part - writing tweets*/
+#write-post {
+    width: 95%;
+    position: absolute;
+    top: 50px;
+    display: grid;
+    grid-template-rows: 60px 40px;
+    padding: 10px;
+}
+
+#write-post input {
+    height: 40px;
+    width: calc(1fr- 100px);
+    border: none;
+}
+
+#write-post input::placeholder {
+    font-size: 18px;
+}
+
+#write-post > div:nth-child(1) {
+    display: grid;
+    grid-template-columns: 50px 1fr;
+}
+
+#post-quick-menu {
+    position: relative;
+    width: 96%;
+    display: grid;
+    height: 32px;
+    grid-template-columns: repeat(5, 40px) minmax(40px, 1fr) 90px;
+}
+#post-quick-menu > div {
+    border: 1px dotted grey;
+}
+
+#post-quick-menu .rounded-container:hover {
+    box-shadow: 2px 2px 5px grey;
+}
+/*actual feed*/
+#feed {
+    border-top: 10px solid rgba(128, 128, 128, 0.264);
+    display: grid;
+    grid-auto-rows: repeat(auto, fit-content);
+    height: calc(100vh - 200px);
+    overflow: auto;
+    position: absolute;
+    top: 170px;
+}
+
+.post {
+    padding: 10px 10px;
+    gap: 5px;
+    display: grid;
+    min-height:200px;
+    grid-template-columns: 40px 1fr;
+    border-bottom: 1px solid grey;
+}
+
+.post-content {
+    display: grid;
+    grid-template-rows: 40px repeat(2, auto-fit) 40px;
+    gap: 5px;
+}
+
+.react {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+}
+
+.react div {
+    border: 1px dotted grey;
+}
+
+/*STYLES THE RIGHT-MENU*/
+
+#right-menu {
+    height: calc(100vh - 20px)
+}
+
+#right-menu > main {
+    height: calc(100vh - 55px);
+    overflow: auto;
+    position: absolute;
+    top: 40px;
+    width: 100%;
+    margin: 10px;
+}
+
+#right-menu > main > div {
+    display: grid;
+    grid-template-rows: 40px repeat(auto, 1fr) ;
+    background-color: rgb(245, 245, 245);
+    border-radius: 5px;
+    padding: 5px;
+    margin: 10px 0;
+}
+
+
+#right-menu > main > div > div {
+    border-bottom: 1px solid grey;
+    height: 100px;
+    padding: 5px;
+}
+
+
+#right-menu input {
+    background-color: rgb(240, 240, 240);
+    border: none;
+}
+/*STYLES THE MESSAGES BOX*
+
+
+
+
+/*FOR BIGGER DEVICES (laptops, tablets,...)*/
+
+    #bottom-menu {
+        width: 0;
+        content-visibility: hidden;
+    }
+
+/*FOR SMALLER RESOLUTION DEVICES*/
+
+@media screen and (max-width:480px) {
+    #left-menu, 
+    #right-menu {
+        padding: 0;
+        margin: 0;
+        width: 0;
+        content-visibility: hidden;
+    }
```

Comment:
Agree with Tom, mobile view looking good.

I'd suggest maybe limiting the amount of icons in the bottom bar and giving them each more space. It could be quite tricky to select them on a small screen.

Reply:
ooops, didn't think of that last bit - makes sense though! I'll keep 3, have 'more' as the 4th, with the assumption that the ones that aren't visible can be accessed  if 'more' is open. 

Could I actually do that last bit (having a 'more' menu be accessible once activated) without needing JS?

Reply:
It's kinda possible in just CSS, for my Discord assignment I used [this thread](https://stackoverflow.com/questions/44832572/pure-css-clickable-dropdown) to make dropdowns.

Reply:
I'll give that a go then, cheers!

Reply:
Found a way, might be kinda similar (mine behaves a bit like a pop up thingy though, and uses :focus (I'm actually not sure how hover behaves with phones, so went for avoiding it)) - not the most elegant, but it seems to work haha